src/main/java/org/aksw/gerbil/evaluate/ExtendedEvaluationResult.java

```diff
@@ -0,0 +1,34 @@
+package org.aksw.gerbil.evaluate;
+
+import java.util.ArrayList;
+import java.util.List;
+public class ExtendedEvaluationResult implements EvaluationResult{
```

Comment:
Please add a Javadoc comments to the class, attributes and methods.

The name of the class is also not very specific. "ExtendedEvaluationResult" could be nearly anything. Maybe we can have a more specific name?

Reply:
Added javadoc comments 

and updated class name to `AggregateContingencyMetricsReport`